dsb-test-util/build.gradle

```diff
@@ -0,0 +1,19 @@
+plugins {
+    id 'java'
+}
+
+dependencies {
+    compile project(":dsb-api")
+}
+group 'com.spectralogic.dsbrowser'
+version '5.0.0'
+
+sourceCompatibility = 1.8
```

Comment:
Most of these items should be inherited by the parent.  The only thing you must specify is the dependencies, but I like to put the inter module dependencies in the root build.gradle file.